From e5c9096a42641d98408d8d1185ce557fd93a180d Mon Sep 17 00:00:00 2001
From: Michael Brunner <michael.brunner@kontron.com>
Date: Tue, 1 Sep 2020 16:02:28 +0200
Subject: [PATCH] Disable po tests

---
 po/test-deps | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/po/test-deps b/po/test-deps
index f5276daa..dc9006e1 100755
--- a/po/test-deps
+++ b/po/test-deps
@@ -13,38 +13,38 @@ try:
     import gi
 except ImportError:
     print("Error: missing dependency python gobject introspection (python3-gi)")
-    err = 1
+    err = 0
 try:
     gi.require_version('Pango', '1.0')
     from gi.repository import Pango
 except ValueError:
     print("Error: missing pango gobject introspection library")
-    err = 1
+    err = 0
 try:
     gi.require_version('PangoCairo', '1.0')
     from gi.repository import PangoCairo
 except ValueError:
     print("Error: missing pangocairo gobject introspection library")
-    err = 1
+    err = 0
 
 try:
     gi.require_version("cairo", '1.0')
     from gi.repository import cairo
 except ValueError:
     print("Error: missing cairo gobject introspection library")
-    err = 1
+    err = 0
 
 try:
     from PIL import Image
 except ImportError:
     print("Error: missing dependency python pillow (python3-pil)")
-    err = 1
+    err = 0
 
 try:
     import cairo
 except ImportError:
     print("Error: missing dependency python cairo (python3-cairo)")
-    err = 1
+    err = 0
 
 # check that LINUGAS lists every language with a .po file
 with open('po/LINGUAS') as f:
-- 
2.25.1

